vendor/bat-native-ads/src/bat/ads/internal/ads_impl.cc

```diff
@@ -260,20 +268,35 @@ void AdsImpl::OnWalletUpdated(const std::string& id, const std::string& seed) {
   account_->SetWallet(id, seed);
 }
 
-void AdsImpl::OnResourceComponentUpdated(const std::string& id) {
-  ResourceManager::GetInstance()->UpdateResource(id);
+void AdsImpl::GetStatementOfAccounts(GetStatementOfAccountsCallback callback) {
+  if (!IsInitialized()) {
+    callback(/* success */ false, {});
+    return;
+  }
+
+  account_->GetStatement(
+      [callback](const bool success, const StatementInfo& statement) {
```

Comment:
Lambda captures are not allowed in chromium. This could be passed as a bound argument into a `base::OnceClosure`